clang/test/dpct/LibCU/libcu_array.cu

```diff
@@ -1,5 +1,5 @@
 // UNSUPPORTED: v7.0, v7.5, v8.0, v9.0, v9.2, v10.0, v10.1, v10.2, v11.0, v11.1
```

Comment:
for disable lit  if CUDA version does not support， we need disable **windows** and linux.
like #539